### DISCUSSION TITLE
Add emoji above profile card?

### DISCUSSION THREAD

**Discussion by GHuserE** [community/community#123118] [New to GitHub] [answered]
How do people add this bar above their profile image? It appears when you hover over their name with the mouse.

![image](https://github.com/community/community/assets/168312322/ebe2ae7e-0021-4fdd-b7e1-f55c35a2313f)

**Comment by davevad93** [comment, marked as answer]
Hi @GHuserE ,

To do so, follow these steps:

1. Click on the little face located on the bottom right side of your profile's picture:
![Captura](https://github.com/community/community/assets/94308234/65015a3f-a778-40a1-9d5c-76aab0d9aec5)
2. Choose your status:
![3](https://github.com/community/community/assets/94308234/76d10ebe-4c59-45b2-aa9c-0ce701e8d6af)

Final result:

![4](https://github.com/community/community/assets/94308234/4fb51bf8-fe1b-4294-ac0a-ed7cebe1d374)

**Reply by GHuserE** [reply]
Thanks a lot for your answer, appreciate it! Cute cat :heart: :heart: @davevad93

**Reply by davevad93** [reply]
You're welcome @GHuserE , have a nice day!
